requirements/prod.txt

```diff
@@ -59,7 +59,6 @@ idna==3.3
     #   snowfakery
 immutables==0.16
     # via
-    #   contextvars
     #   snowfakery
```

Comment:
I suspect immutables will go away once we update to the version of snowfakery that no longer includes contextvars.